### PR TITLE
Initial cut of Docs

### PR DESCRIPTION
**Pull request by karianna**

### CODE REVIEW

**Review by gdams — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,16 +1,56 @@
...
+# Checkout the main branch
+git checkout main
+./dockerhub_doc_config_update.sh
+```
+
+This script will create a file called _eclipse-temurin_.
+
+Then edit the [Manifest on Docker Hub](https://github.com/docker-library/official-images/blob/master/library/eclipse-temurin), replacing all of the contents.
+
+At the bottom of that edit screen add a title and description for the commit and click on hte _Propose Change_ button.
```
Comment (timja) [inline README.md:48]:
```suggestion
At the bottom of that edit screen add a title and description for the commit and click on the _Propose Change_ button.
```

**Review by sxa — COMMENTED** [review]
Do we have any sort of overview of what the two scripts `update_all.sh` and perhaps also the manifest update one do? Ideally we need something like "The scripts checks for X,Y,Z and updates A,B,C". This is useful to know when the process needs to be run.

Might also be useful to flesh out the "diff output created"  could be fleshed out - is that just a normal review of what's in the PR?

**Comment by tellison** [comment]
ECA check fails on [anonymous gdams](https://github.com/adoptium/containers/pull/67/commits/d1cb554b4b02fe11d88bfa04f0461f59d05c636a) change

File: README.md [hunk 2]
```diff
@@ -53,4 +61,6 @@ Once that PR has been created it will be automatically tested and reviewed by Do
 
 ### Diff Output at Docker Hub
 
-It can be useful to look at the diff output created by one of the Docker Hub GitHub Actions on the Pull Request.
+It can be useful to look at the diff output created by one of the Docker Hub GitHub Actions on the Pull Request. This output
+should not be read aas a traditional PR (since Docker Hub bots do move things around, so you may see what looks like odd deletions)
```
Comment (tellison) [inline README.md:65]: typo "aas" -> "as"

File: README.md [hunk 3]
```diff
@@ -53,4 +61,6 @@ Once that PR has been created it will be automatically tested and reviewed by Do
...
+but as a sanity check to make sure you see the platoforms/architectures that you expect.
```
Comment (tellison) [inline README.md:66]: typo "platoforms" -> "platforms"

File: README.md (hunk 2 above)
Comment (karianna) [reply README.md]: Fixed thanks

File: README.md (hunk 3 above)
Comment (karianna) [reply README.md]: Fixed thanks.

Comment (sxa) [inline README.md]: Hmmm github isn't giving me the option for suggesting changes on this, but typo here "platoforms". Once that's addressed (and the ECA check failure resolved) I'm happy with this now :-)

**Review by sxa — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (sxa) [reply README.md]: Ah ignore that - I see it was changed in a subsequent commit.

**Review by sxa — APPROVED** [review]
LGTM but the ECA check failure should be addressed prior to merging

**Pull request merged**